prog/minio/minio_server_nexus.rb

```diff
@@ -39,7 +39,7 @@ def self.assemble(minio_pool_id, index)
 
   def before_run
     when_destroy_set? do
-      if strand.label != "destroy"
+      if strand.label != "destroy" && strand.label != "minio_restart"
```

Comment:
I think we should pop the stack frame in this case. Server might be in unrestartable condition, which should not block destroy operation.

We could have a while loop until we have one frame in the stack.

Reply:
I'll fix this one on top of this PR. For now, I'll remove it.